pkg/proxy/metrics/metrics.go

```diff
@@ -138,6 +138,17 @@ var (
 		},
 	)
 
+	// IptablesCTStateInvalidDroppedPackets is the number of packets dropped
+	// by iptables which were marked INVALID by conntrack.
+	IptablesCTStateInvalidDroppedPackets = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Subsystem:      kubeProxySubsystem,
+			Name:           "iptables_ct_state_invalid_dropped_packets",
+			Help:           "packets dropped by iptables to work around conntrack problems",
+			StabilityLevel: metrics.ALPHA,
+		},
+	)
+
```

Comment:
Why isn't this a counter?

Reply:
IIRC counter only allows `Inc()` and `Add()`, we are not maintaining the actual number of packets in the memory, we just read the number of packets from the `nfacct` sub-system and set the metric with that value.

Reply:
You will need to implement the Collector interface, and use NewConstMetric to provide all the current values for the counter.  See https://github.com/kubernetes/kubernetes/blob/58bbf6991303fa97660beafc74d0d2d6e2c1c623/staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go#L303 for an example.

Reply:
I have added the custom metric.